book/src/pong-tutorial/pong-tutorial-03.md

```diff
@@ -169,7 +223,28 @@ If we run the game now, we'll see the console print our keypresses. Let's
 make it update the position of the paddle. To do this, we'll modify the y
 component of the transform's translation.
 
-```rust,ignore
+```rust,no_run,noplaypen
+# extern crate amethyst;
+# use amethyst::core::Transform;
+# use amethyst::ecs::{Join, Read, ReadStorage, System, WriteStorage};
+# use amethyst::input::InputHandler;
+# enum Side {
+#   Left,
+#   Right,
+# }
+# pub struct Paddle {
+#   side: Side,
+# }
+# impl amethyst::ecs::Component for Paddle {
+#   type Storage = amethyst::ecs::VecStorage<Paddle>;    
+# }
+# pub struct PaddleSystem;
+# impl<'s> System<'s> for PaddleSystem {
+#  type SystemData = (
+#    WriteStorage<'s, Transform>,
+#    ReadStorage<'s, Paddle>,
+#    Read<'s, InputHandler<String, String>>,
+#  );
   fn run(&mut self, (mut transforms, paddles, input): Self::SystemData) {
```

Comment:
I don't see where you show where you define (visibly from the user) the System here. Mind pointing it out to me? thanks

Reply:
L114